account_move_sent_usability/views/account_move.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2015-2018 Jacques-Etienne Baudoux (BCIM) <je@bcim.be>
+     Copyright 2023 ACSONE SA/NV
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="view_account_invoice_filter" model="ir.ui.view">
+        <field name="name">account.move.filter</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_account_invoice_filter" />
+        <field name="arch" type="xml">
+
+            <xpath expr="//filter[@name='open']/preceding::separator" position="before">
+                <separator />
+                <filter
+                    name="unsent"
+                    string="Not Sent"
+                    domain="[('is_move_sent', '=', False)]"
```

Comment:
```suggestion
                    domain="[('is_move_sent', '=', False), ('state', '!=', 'cancel')]"
```

I suggest to not show cancelled invoice in this "not sent" filter, as this unnecessarily pollutes the result with invoices that will never be sent.

Reply:
Or not, it's easy enough to click on the Draft and Posted filters in addition to Not Sent